addons/website_slides_survey/models/slide_slide.py

```diff
@@ -8,29 +8,29 @@ class SlidePartnerRelation(models.Model):
     _inherit = 'slide.slide.partner'
 
     user_input_ids = fields.One2many('survey.user_input', 'slide_partner_id', 'Certification attempts')
-    survey_quizz_passed = fields.Boolean('Certification Quizz Passed', compute='_compute_survey_quizz_passed', store=True)
+    survey_scoring_success = fields.Boolean('Certification Quizz Passed', compute='_compute_survey_scoring_success', store=True)
 
-    @api.depends('partner_id', 'user_input_ids.quizz_passed')
-    def _compute_survey_quizz_passed(self):
+    @api.depends('partner_id', 'user_input_ids.scoring_success')
+    def _compute_survey_scoring_success(self):
         passed_user_inputs = self.env['survey.user_input'].sudo().search([
             ('slide_partner_id', 'in', self.ids),
-            ('quizz_passed', '=', True)
+            ('scoring_success', '=', True)
         ])
         passed_slide_partners = passed_user_inputs.mapped('slide_partner_id')
```

Comment:
Renamed to `successed_user_inputs` and `successed_slide_partners` ?

It was "passed" because of quizz_passed, now it does not make much sense anymore.